CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -751,15 +754,15 @@ private PokerAction GetUserAction()
                 _ => PokerAction.CHECK
             };
         }
-        
+
         private PokerAction GetAIAction(int playerIndex)
```

Comment:
still missing summary comments
the wall of comments within this method should be within the summary comments